### DISCUSSION TITLE
See entered search term

### DISCUSSION THREAD

**Discussion by on3iro** [dj95/zjstatus#85] [Q&A] [answered]
Hey,

is there currently a way to see the typed in search term in EnterSearch-Mode?

Thanks :)

**Comment by dj95** [comment, marked as answer]
Hey,

the search term should be available in the pane frames. There's extra a feature implemented, that will show the frame on search, even it was hidden :)

**Reply by on3iro** [reply]
ah thanks for the quick reply. Unfortunately I almost always have these frames disabled. It would be cool if there was a widget for this (if that's currently even possible).

**Reply by dj95** [reply]
I think that's not really possible, unfortunately, as zjstatus only receives keyboard input, that is directly directed to it (when it would be focused, so basically none :D).

There's also no real way to extract the search term.

I could offer to implement a feature, that hides the pane frames except for search mode, such that you can see the search term. Would that help here?

**Reply by on3iro** [reply]
That would be pretty cool, yeah - so I wouldn't have to do that manually <3

**Reply by dj95** [reply]
It is implemented now :D but it'll be released as soon as the new zellij version arrives, as I'm waiting for some upstream fixes to finally finish the "toggle statusbar" functionality.

Feel free to build the new version on your own, such that you can use it. The config parameter is named `hide_frame_except_for_search` and must be set to `"true"`.

However, I'm not sure if you need to enable frames in the global zellij config for this to work. I've them enabled either way. But zjstatus will hide them for you :D
